Automattic/jetpack
# Jetpack: Jetpack Forms as patterns

This PR will register "Forms" patterns category and all Jetpack Form variations as patterns on that category. 

This provides both more visibility for those blocks as well as previews, something not available right now with the variation picker.

This PR registers the patterns through the PHP Blocks API, but it should be interesting to try the same from JS.

#### Changes proposed in this Pull Request:
* Add/register "Forms" patterns category
* Register all Jetpack Form variations as patterns

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
Open the inserter, go to the Patterns tab and select "Forms" category. You should see all the Jetpack Form variations listed there. Insert on post to verify they all work as expected.

<img width="348" alt="image" src="https://user-images.githubusercontent.com/157240/197581828-712a9e32-318b-4653-95b4-2da1e330bf42.png">
